### ISSUE TITLE
the running time was so long that i couldn't finish my task

### ISSUE THREAD

**Issue by xinkaitong** [bioXiaoheng/BallerMixPlus#3]
Hi Xiaoheng,
I have read your paper, well done! Now, the software has been installed and can be executed without mistakes. But it spent much time executing the example file which contained 758 variants. Am I wrong? I have about 40 million variants to calculate. If so, it's impossible to finish my task.

**Comment by bioXiaoheng** [comment]
Hi there,

Thanks for trying out the software! Which platform/system environment are you running it on? My experience is that the example typically shouldn't take more than 30min on a regular linux machine, but if you run it on window's powershell or its linux sub-system (WSL) with conda environment, somehow python would become painstakingly slow and the task would take hours.

**Comment by xinkaitong** [comment]
> Hi there,
> 
> Thanks for trying out the software! Which platform/system environment are you running it on? My experience is that the example typically shouldn't take more than 30min on a regular linux machine, but if you run it on window's powershell or its linux sub-system (WSL) with conda environment, somehow python would become painstakingly slow and the task would take hours.

yes, I used a regular linux machine, but it actually took me much time. Another answer I want to know is that how can I finish my task over 40 million variants using your software.

**Comment by bioXiaoheng** [comment]
> > Hi there,
> > Thanks for trying out the software! Which platform/system environment are you running it on? My experience is that the example typically shouldn't take more than 30min on a regular linux machine, but if you run it on window's powershell or its linux sub-system (WSL) with conda environment, somehow python would become painstakingly slow and the task would take hours.
> 
> yes, I used a regular linux machine, but it actually took me much time. Another answer I want to know is that how can I finish my task over 40 million variants using your software.

You can consider parallelizing your analyses. You can find more details at issue #1
